### ISSUE TITLE
Slider is not operatable with the keyboard

### ISSUE THREAD

**Issue by maracuja-juice** [designmodo/Flat-UI#245]
The sliders in this project are not controllable with a keyboard. 
This is very bad for accessibility. Mozilla says about sliders:

> The slider should be keyboard focusable and operable. When the user tabs focus to the slider, it should land on the thumb: the control a mouse user would drag. Arrow keys should operate as follows (localization for right-to-left languages should reverse the direction of the arrows):

Key(s) | Action
-- | --
Right and Up arrows | Increase the selected value
Left and Down arrows | Decrease the selected value
Page Up and Page Down | Optionally increase and decrease the value by a set amount (e.g. by 10 on a range from 0 to 100)

But this is not possible with your slider elements. ([Source](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_slider_role))

**Comment by maracuja-juice** [comment]
I was able to fix this for my own site: I added `tabindex=" 0"` on the Slider handle which made it possible to select it with the tab key. The left/right/up/down arrow keys are already supported by the JQuery UI slider.

**Comment by maracuja-juice** [comment]
Although I fixed this for myself: I think this should be added to the Flat-UI kit aswell. What do you think @StefanYohansson ?

**Comment by StefanYohansson** [comment]
yeah, accessibility is a great idea. I suggest that if you are able to help Flat UI project, you should email @MonsterCritic
